stable/mssql-linux/templates/pvc.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingDataClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "mssql.fullname" . }}-data
+  labels:
+    app: {{ template "mssql.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+{{- if .Values.persistence.annotations }}
+  annotations:
+{{ toYaml .Values.persistence.annotations | indent 4 }}
+{{- end }}
+spec:
+  accessModes:
+    - {{ .Values.persistence.dataAccessMode | quote }}
+  resources:
+    requests:
+      storage: {{ .Values.persistence.dataSize | quote }}
+{{- if .Values.persistence.storageClass }}
+{{- if (eq "-" .Values.persistence.storageClass) }}
+  storageClassName: ""
+{{- else }}
+  storageClassName: "{{ .Values.persistence.storageClass }}"
+{{- end }}
+{{- end }}
+{{- end -}}
+---
```

Comment:
Please create separate files per resource.

Reply:
This disk is for the LDF files that are part of a MSSQL database (transaction log file).  It is common practices in the SQL Server realm to separate out mdf files (master data file) and ldf files (transaction log files) for performance reasons.
https://docs.microsoft.com/en-us/sql/relational-databases/policy-based-management/place-data-and-log-files-on-separate-drives

SQL Server has a separate log called ERRORLOG that is stdout as part of the image.

However - I can see there might be confusion in the labeling of the disks.  Do you have any thoughts on labeling?

I'll also create separate files for each.

Reply:
I went ahead and created separate files for each resource and spelled out the name of the transaction log instead of a generic "log" labeling.